### PR TITLE
Feedback properties

### PR DESCRIPTION
**Pull request by higs4281**
Add Feedback model properties to assist in analysis

This addresses GHE paying-for-college/dynamic-disclosures-dev issue 226

The PR also makes some minor script fixes that address #298:
- A bug in the `update_colleges` script that caused grad_rates to fail to update

## Additions
- Four model-method properties for the Feedback model
- tests for the new methods

### CODE REVIEW

File: paying_for_college/models.py [hunk 1]
```diff
@@ -739,6 +739,65 @@ class Feedback(models.Model):
...
+    @property
+    def school(self):
+        """Returns a school object, derived from a feedback url"""
+        if not self.url:
+            return ''
+        row = self.parsed_url
+        if row and row.get('iped'):
+            return School.objects.get(pk=row['iped'])
+        else:
+            return ''
```
Comment (willbarton) [inline paying_for_college/models.py]: Is a `''` return here (and line 758) appropriate in place of a `School`? I don't see right off where this is called, so I'm not sure.

File: paying_for_college/models.py [hunk 2]
```diff
@@ -739,6 +739,65 @@ class Feedback(models.Model):
...
+    def unmet_cost(self):
+        """Calculates and returns a disclosure's unmet cost"""
+        url_data = self.parsed_url
+        if not url_data:
+            return None
+
+        def total_fields(field_list):
+            total = 0
+            for field in field_list:
+                if field in url_data.keys() and url_data[field] != '':
```
Comment (willbarton) [inline paying_for_college/models.py]: `if url_data.get('field', '') != ''` might be simpler here.

**Review by willbarton — APPROVED** [review]
Looks good to me overall. Nice style cleanup. Lack of familiarity with the code means I'm going over this with a fine-toothed nit picker, but neither of my comments are blockers.

**Comment by higs4281** [comment]
@willbarton Thanks for the close review. suggestions taken!

**Pull request merged**